client/src/dialogs/feedback/AnythingElseDialog.js

```diff
@@ -42,16 +78,36 @@ const AnythingElseDialog = ({ onClose, onSubmitClick }) => {
       aria-labelledby="customized-dialog-title"
       open={openDialog}
     >
+      <Snackbar
+        open={openPleaseFillSnackbar}
+        autoHideDuration={6000}
+        onClose={handlePleaseFillSnackbarClose}
+      >
+        <Alert onClose={handlePleaseFillSnackbarClose} severity="error">
+          Please enter review
+        </Alert>
+      </Snackbar>
+      <Snackbar
+        open={openSaveErrorSnackbar}
+        autoHideDuration={6000}
+        onClose={handleSaveErrorSnackbarClose}
+      >
+        <Alert onClose={handleSaveErrorSnackbarClose} severity="error">
+          An error occured. Please try again
```

Comment:
If we have this text as a state variable, then we only need one snackbar component, and the code is more extendable